lib/View/BatchDeleteView.js

```diff
@@ -5,6 +5,7 @@ class BatchDeleteView extends View {
         super(name);
 
         this._type = 'BatchDeleteView';
+        this._enabled = true;
```

Comment:
On the contrary, it fix thing. This view does not need field, and if none were added to it, a view is not enabled error is thrown.